### PR TITLE
Show progress bar on entry listing

### PR DESCRIPTION
**Pull request by kevinkace**
Two things with this:
1. The contentHd/pagination won't show until listing is loaded.  
   - `layout` component could take 2 children: `contentHd` and `contentBd`
   - `layout` component could look for a `loading` flag
   - components could render the progress bar themselves
2. Stylistically what do you think about `!condition ? null : m("div")` vs `condition ? m("div") : null`? Reminds me of the early out approach, also no dangling `null`s.

### CODE REVIEW

**Comment by ctcleary** [comment]
Re # 2: I keep going back and forth on that. Because we expect positive conditionals rather than negative, and I already ran into an issue where I did the reverse-conditional for the early `null` but it made it really hard to come back to because my conditions were doing the opposite of my instinct. This is one of those thing I really dislike about these huge mithril objects but apart from pulling these instances out into their own function (which @tivac really dislikes for reasons I do understand), I haven't found a solution I like.

**Comment by ctcleary** [comment]
Seems sane. 👍

**Comment by kevinkace** [comment]
Oh wait, I now see that if there are no posts (ie a new schema) then the list never loads.

**Comment by ctcleary** [comment]
In that case, I retract my endorsement of sanity.

File: src/pages/listing/index.js [hunk 1]
```diff
@@ -308,9 +311,9 @@ export function view(ctrl) {
     }
 
     return m.component(layout, {
-        title   : get(ctrl, "schema.name") || "...",
-        content : [
-
+        title     : get(ctrl, "schema.name") || "...",
+        inProgress : ctrl.inProgress,
+        content   : [
```
Comment (kevinkace) [inline src/pages/listing/index.js]: Whitespace again.

I need to get eslint working on my laptop again.

**Comment by kevinkace** [comment]
Now updated to use `options.inProgress`. Thoughts?

**Comment by tivac** [comment]
Rename `inProgress` to `loading` please, easier to type.

I prefer the trailing `: null`, maybe I'm just being weird?

**Comment by kevinkace** [comment]
What is happening...? A merge conflict showed up (for no reason that I can see), so I updated my forks master with this master, then rebased on my forks branch, and now all commits are duplicated?!? Is this from replaying my commits after the rebase?

I also cannot explain why there are changes shown to ` src/pages/content-edit/content-state.js`, it looks to match the state on master...

**Comment by tivac** [comment]
In theory it'll come out in the merge wash. I'm gonna try squashing & merging locally so I can verify instead of pressing the handy button and fucking it all up.

**Comment by tivac** [comment]
Manually merged by https://github.com/tivac/crucible/commit/697a0f9e03eb446eae4ed22e05fe13f827fc1e7f

**Comment by kevinkace** [comment]
Thanks.

**Comment by tivac** [comment]
👌

**Pull request closed without merging**